boards/arm/frdm_k82f/frdm_k82f.dts

```diff
@@ -224,3 +224,11 @@
 &pit0 {
 	status = "okay";
 };
+
+arduino_i2c: &i2c0 {
+	status = "okay";
+};
+
+arduino_spi: &spi0 {
+	status = "okay";
```

Comment:
Please add `pinctrl-0` similar to commit 9901871736c85a044797a645fc617b6c54a2e288